src/components/Menu.jsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faTrashAlt } from '@fortawesome/free-solid-svg-icons';
+import Rating from './Rating';
+
+const Menu = () => {
+    const [restaurants, setRestaurants] = useState([{name:'',location:'',rating:''}]);
+    
+    const handleSubmit = (name, location, rating) => {
+            name = document.getElementById('restaurant-name').value;
```

Comment:
we don't want to use this syntax in React please figure out another way to do it for all inputs